core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -307,7 +308,7 @@ private DataType fromCassandraDataType(
       case BLOB:
         return com.datastax.driver.core.DataType.blob();
       default:
-        throw new UnsupportedOperationException(String.format("%s is not yet supported", dataType));
+        throw new AssertionError();
```

Comment:
`dataType` is mostly given from users?
If so, should it be an unchecked exception?

Reply:
I understand the point. Thank you!